tld-awesome-click-to-tweet.php

```diff
@@ -29,7 +29,9 @@ function tld_actt_load_intents_assets() {
 	wp_enqueue_style( 'tld-titillium-font', '//fonts.googleapis.com/css?family=Titillium+Web' );
 	wp_enqueue_style( 'tld-alegreya-font', '//fonts.googleapis.com/css?family=Poiret+One' );
 }
-add_action( 'wp_enqueue_scripts', 'tld_actt_load_intents_assets' );
+add_action( 'wp_enqueue_scripts', 'tld_actt_load_intents_assets' ); // For Frontend
+add_action( 'enqueue_block_editor_assets', 'tld_actt_load_intents_assets' ); // For Backend Block
```

Comment:
@ajitbohra `enqueue_block_editor_assets` action is only available if Gutenberg is installed, so we can check before hooking to action?

Reply:
MIssed it we need check here will add that